src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/BindToInputBindingProvider.cs

```diff
@@ -46,9 +46,7 @@ public Task<IBinding> TryCreateAsync(BindingProviderContext context)
                 return Task.FromResult<IBinding>(null);
             }
 
-            var type = typeof(ExactBinding<>).MakeGenericType(typeof(TAttribute), typeof(TType), typeUser);
-            var method = type.GetMethod("TryBuild", BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public);
-            var binding = BindingFactoryHelpers.MethodInvoke<IBinding>(method, this, context);
```

Comment:
Before, ConverterMAnager only had <T> overloads, so you had to do this MakeGenericType to call it with a runtime type. Now, It has loose signarures that just take a Type, so it's easier to call without generics.